### DISCUSSION TITLE
O que são métodos HTTP?

### DISCUSSION THREAD

**Discussion by ameninadogorro** [community/community#36456] [Apps, API and Webhooks] [answered]
Gostaria de mais definições sobre os métodos HTTP, pois fiquei um pouco confusa em relação a eles.

**Comment by pedrohgmuniz** [comment, marked as answer]
O protocolo HTTP define um conjunto de métodos de requisição responsáveis por indicar a ação a ser executada para um dado recurso. Esses métodos podem ser descritos como substantivos mas também como HTTP Verbs. Nas requisições, especificamos o que chamamos de método HTTP ou verbo.

**Comment by ettaboyle** [comment]
Thanks for participating!

We take our Code of Conduct very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.

**Comment by ayyanaliwagi88** [comment]
- - [ ] []()Ok sain
